spec/board_spec.rb

```diff
@@ -84,4 +85,23 @@
       expect(board.valid_placement?(submarine, ["A1", "B1"])).to be false
     end
   end
+
+  context 'Rendering the board' do
```

Comment:
Printing returns nil so this test wont pass. We could `expect(board.render).to be nil` maybe? Or something else if you can think of something better 😅

Reply:
These tests are disabled for now. I'd ideally like the render method to return the board but I haven't quite figured that out yet

Reply:
I think to do that you would have to copy the whole method and put it at the end of the method without the `print`